db/DB.js

```diff
@@ -419,6 +419,30 @@ const DB = {
     await knex('email_addresses').where({ subscriber_id: uid }).del()
   },
 
+  async updateBreachStats (id, stats) {
+    await knex('subscribers')
+      .where('id', id)
+      .update({
+        breach_stats: stats
+      })
+  },
+
+  async getBreachStats (id) {
+    const { breach_stats: breachStats } = await knex('subscribers')
+      .where({ id })
+      .first('breach_stats')
+
+    return breachStats
+  },
```

Comment:
suggestion (non-blocking): we shouldn't need this DB query. In both of the places this is called, the same data is available at `req.user.breach_stats`. (`requireSessionUser` and `requireAdminUser` both set `req.user` to the result of `DB.getSubscriberById`)

Reply:
great call!